settings.gradle

```diff
@@ -87,3 +87,5 @@ include 'eventmesh-webhook:eventmesh-webhook-api'
 include 'eventmesh-webhook:eventmesh-webhook-admin'
 include 'eventmesh-webhook:eventmesh-webhook-receive'
 
+include 'eventmesh-sdks'
+include 'eventmesh-sdk-java'
```

Comment:
Does this mean that SDK should be added into Runtime jar package?

Reply:
This change is indeed unnecessary, reverted.